FORM_Entrainement.Altis/functions/CCOM_PO/fn_CCOM_initPO.sqf

```diff
@@ -0,0 +1,22 @@
+// The source object is most likely the castle tower object (Land_Castle_01_tower_F), from which the played has to rappel down.
```

Comment:
I would place the course over a flat concrete surface to reduce land disparity

Reply:
I thought about that indeed, but since the overall result was quite OK for the two instances of the obstacle course on the current training mission, I didn't do it. Addind a concrete surface underneath might be a little bit more complex, and the result might be weird in some places.

I'll think about it and do a few tests.

Reply:
Up to you whether you prefer to:
* keep this conversation open
* move to a new issue and merge this PR

Reply:
We can merge this "as-is". Concrete surface was part of the issue #3 that will be closed with this PR. We can re open it after merge, in order to specify the need of a comprehensive concrete surface depending on terrain variations and stuff